tox.ini

```diff
@@ -1,26 +1,121 @@
 [tox]
-envlist = py26,py27,py34,py35,py36,pypy,coverage-report,packaging
+
+envlist =
+    test-py{26,27,34,35,36,37,py,py3}
+    coverage_report
+    packaging
+
+skip_missing_interpreters = {tty:True:False}
```

Comment:
Didn't know this was a feature, nice. Syntax for the value tripped me out for a sec, but not bad. :)

Reply:
Yeah, in Klein I used this plus a wrapper script:
```python
skip_missing_interpreters = {env:TOX_SKIP_MISSING_INTERPRETERS:True}
```
But then I saw they added this `tty` thing, and that seems even better.